static/500.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+  <!-- Basic Page Needs
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <meta charset="utf-8">
+  <title>The Gazelle</title>
+
+  <!-- Mobile Specific Metas
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <meta name="viewport" content="initial-scale=1, maximum-scale=1">
+
+  <!-- FONT
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,700" rel="stylesheet">
+
+  <!-- Style
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <style>
+  .fontErr {
+    text-align: center; 
+    vertical-align: middle;
+    line-height: 0.5rem; 
+
+
+  }
+  .centered {
+    text-align: center; 
+    margin-left: auto; 
+    margin-right: auto; 
+  }
+
+  .container {
+  position: relative;
+  width: 100%;
+  max-width: 960px;
+  margin: 0 auto;
+  padding: 0 10px;
+  box-sizing: border-box; }
+
+   .headerLogo {
+    margin-left: auto; 
+    margin-right: auto;
+    width: 8%;
```

Comment:
Having the width at 8% means that on small screens the logo will be tiny. Change this to `px` or adjust with media queries so it works on all device sizes.